CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.13.0 Release
+- Release Date: October 16, 2018
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported
+- Mobile Versions Supported: Mobile app version 1.13+ is required for servers 5.4+
+
+### Highlights
+
+#### View Reactions
+- Added ability to view who reacted to a post.
+
+#### Dropdown menus
+- Added support for message dropdown menus.
+
+#### Hashtags
+- Added support for hashtags.
+
+### Improvements
+- Added support for iPhone XR, XS and XS Max.
+- Added support for nicknames on user profile.
+- On servers 5.4+, added support for searching in direct and group message channels using the "in:" modifier.
+- Channel autocomplete now gets closed if multiple tildes are typed.
+- Added a draft icon in sidebar and channel switcher for channels with unsent messages.
+- Added paging for elasticsearch results.
```

Comment:
Paging for search, the app does not discriminate, perhaps the server does bjt we don’t care?